### PR TITLE
Add map annotation tool

### LINKED ISSUE #4: Add a map annotation tool
The tool should be able to: 
* create navigation goals (poses) 

Part of this issue was originally found here https://github.com/b-it-bots/map_annotation_tool/issues/2

### PR DESCRIPTION
**Pull request by argenos**
Closes #4

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
 
-  <maintainer email="frederik.hegger@h-brs.de">Frederik Hegger</maintainer>
-  <maintainer email="olima_84@yahoo.com">Oscar Lima</maintainer>
+  <maintainer email="robotics@mail.inf.h-brs.de">MAS Robotics</maintainer>
 
   <license>GPLv3</license>
 
   <author email="frederik.hegger@h-brs.de">Frederik Hegger</author>
   <author email="olima_84@yahoo.com">Oscar Lima</author> <!--path length calculator-->
+  <author email="suvich15@todo.todo">Sushant Chavan</author>
```
Comment (Sushant-Chavan) [inline package.xml]: Could you please change the email to `sushant.chavan@smail.inf.h-brs.de`. I think we forgot about this when we released the tool.

**Review by Sushant-Chavan — APPROVED** [review]
Just one small change in the author's email which is result of my mistake during the first release of the tool.

File: package.xml (hunk 1 above)
Comment (argenos) [inline package.xml]:
```suggestion
  <author email="sushant.chavan@smail.inf.h-brs.de">Sushant Chavan</author>
```

**Review by alex-mitrevski — APPROVED** [review]

**Review by sthoduka — APPROVED** [review]
I just added Ahmed to the authors. Looks fine otherwise

**Comment by argenos** [comment]
What about the license in the header files? Are we using stuff from Willow Garage?

**Comment by Sushant-Chavan** [comment]
> What about the license in the header files? Are we using stuff from Willow Garage?

I have opened the PR #6  to update the licence.

**Pull request merged**